### PR TITLE
EventGrid: Fix help text to point to the right endpoint parameter in Event Subscriptions commands.

### PR DESCRIPTION
**Pull request by ahamad-MS**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [X] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

File: src/azure-cli/HISTORY.rst [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 Release History
 ===============
+**EventGrid**
```
[the remark this answers is not in the document]
Comment (ahamad-MS) [reply src/azure-cli/HISTORY.rst]: @yugangw-msft  Thanks. Fixed.

[the remark this answers is not in the document]
Comment (ahamad-MS) [reply src/azure-cli/HISTORY.rst]: @yugangw-msft Can you please approve/merge this change? thanks

[the remark this answers is not in the document]
Comment (yugangw-msft) [reply src/azure-cli/HISTORY.rst]: Sorry for the delay which caused new conflicts. I am opening a new one with the change ported and will merge soon.

**Comment by yugangw-msft** [comment]
#10540 should take care of it

**Pull request closed without merging**